### ISSUE TITLE
FTP server connection not showing files

### ISSUE THREAD

**Issue by nv17x** [rashevskyv/dbi#351]
Hello, 

I setup a connection to my NAS through FTP but when i connect dbi doesn't show files ("This view contains no item"). What could be the issue? I can access to nas files through other Switch apps like Tinfoil.
Thank you.

Have a good day.

**Comment by Furymaxime** [comment]
Bonjour tu a trouvé la solution ? J'ia le même problème :(

**Comment by nv17x** [comment]
No, I didn't find any solution and I ended up using Tinfoil (way worst  and slow than dbi).

**Comment by Furymaxime** [comment]
Merci de ta réponse. Tu a testé aussi avec un Nas Synology ?

**Comment by nv17x** [comment]
No, I have only a Fritz Nas.
